### ISSUE TITLE
Complete help text with `--help`?

### ISSUE THREAD

**Issue by olgabot** [dib-lab/sourmash#706]
Using [this Dockerfile](https://github.com/nf-core/kmermaid/blob/master/Dockerfile), I'm unable to get the full help text with the descriptions so nicely written in [commands.py](https://github.com/dib-lab/sourmash/blob/master/sourmash/commands.py). No arguments, `-h` and `--help` all output the same help text, whereas I would expect `--help` to show the full description. Do you know why this is? Thank you!

```
(base) root@431eec5dfd0f:/home# sourmash compute -h
usage: sourmash [--protein] [--no-protein] [--dayhoff] [--no-dayhoff] [--dna]
                [--no-dna] [-q] [--input-is-protein] [-k KSIZES]
                [-n NUM_HASHES] [--check-sequence] [-f] [-o OUTPUT]
                [--singleton] [--merge MERGED] [--name-from-first]
                [--input-is-10x] [-p PROCESSES] [--track-abundance]
                [--scaled SCALED] [--seed SEED] [--randomize]
                [--license LICENSE]
                filenames [filenames ...]
sourmash: error: the following arguments are required: filenames
(base) root@431eec5dfd0f:/home# sourmash compute --help
usage: sourmash [--protein] [--no-protein] [--dayhoff] [--no-dayhoff] [--dna]
                [--no-dna] [-q] [--input-is-protein] [-k KSIZES]
                [-n NUM_HASHES] [--check-sequence] [-f] [-o OUTPUT]
                [--singleton] [--merge MERGED] [--name-from-first]
                [--input-is-10x] [-p PROCESSES] [--track-abundance]
                [--scaled SCALED] [--seed SEED] [--randomize]
                [--license LICENSE]
                filenames [filenames ...]
sourmash: error: the following arguments are required: filenames
(base) root@431eec5dfd0f:/home# sourmash compute
usage: sourmash [--protein] [--no-protein] [--dayhoff] [--no-dayhoff] [--dna]
                [--no-dna] [-q] [--input-is-protein] [-k KSIZES]
                [-n NUM_HASHES] [--check-sequence] [-f] [-o OUTPUT]
                [--singleton] [--merge MERGED] [--name-from-first]
                [--input-is-10x] [-p PROCESSES] [--track-abundance]
                [--scaled SCALED] [--seed SEED] [--randomize]
                [--license LICENSE]
                filenames [filenames ...]
```

**Comment by luizirber** [comment]
ugh, this is because we don't do [proper argparse subcommands](https://github.com/dib-lab/sourmash/blob/0d69b7973e1b4e690933b27aeb8d11becc876e19/sourmash/__main__.py#L68)... There is some discussion in https://github.com/dib-lab/sourmash/pull/245#pullrequestreview-39344393 on how to do this

**Comment by ctb** [comment]
I think this is much simpler now that #785 is merged!

**Comment by ctb** [comment]
I believe this was fixed in #785!
